test-clients/src/main/java/com/hedera/services/bdd/spec/queries/meta/HapiGetTxnRecord.java

```diff
@@ -179,11 +182,10 @@ public HapiGetTxnRecord hasChildRecordCount(int count) {
 		return this;
 	}
 
-	public HapiGetTxnRecord hasAliasInChildRecord(final String validAlias, int num) {
+	public HapiGetTxnRecord hasAliasInChildRecord(final String aliasingKey, final int childIndex) {
 		requestChildRecords = true;
 		hasAliasInChildRecord = true;
```

Comment:
we can remove the field `hasAliasInChildRecord` now

Reply:
Good point, removed!